bears/vcs/git/GitCommitBear.py

```diff
@@ -77,6 +51,13 @@ def get_metadata(cls):
             cls.get_body_checks_metadata(),
             cls.get_issue_checks_metadata())
 
+    @classmethod
+    def check_prerequisites(cls):
```

Comment:
why did you move this method?  It was before `get_shortlog_checks_metadata`

Reply:
I must've missed it during the shifting of code
Will fix it :smile: